test/TestProjects/MgmtTypeSpec/src/Generated/Extensions/MgmtTypeSpecExtensions.cs

```diff
@@ -151,7 +151,7 @@ public static Response<FooResource> GetFoo(this ResourceGroupResource resourceGr
         /// </item>
         /// <item>
         /// <term>Operation Id</term>
-        /// <description>PrivateLinkResource_GetAllPrivateLinkResources</description>
+        /// <description>PrivateLinks_listByMongoCluster</description>
```

Comment:
here is the difference between original name and `@clientName`: https://github.com/Azure/autorest.csharp/blob/371c2a2029ae8a2baef218b9b55798433f4b7b75/test/TestProjects/MgmtTypeSpec/PrivateEndpointConnection.tsp#L19-L20